### ISSUE TITLE
tags on Features page are missing context

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6792]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `enabled` and `disabled` tags on the Features page are missing context and do not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin user, go to https://dev.nurax.samvera.org/admin/features?locale=en and run the Site Improve browser extension

![features_table](https://github.com/samvera/hyrax/assets/32644254/3caafa60-2853-43f1-a841-cec663956f8f)

**Comment by no-reply** [comment]
could this whole column just be dropped? i'm not sure i understand it's function if the ON/OFF gives the current status.

**Comment by christhepianist** [comment]
Tested and confirmed.

**Comment by nhomenda** [comment]
Tested; confirmed this Site Improve error does not occur on dev or pg.
